tests/backstop_data/backstop.tpl.json

```diff
@@ -17,9 +17,12 @@
   "report": ["CLI", "browser"],
   "engine": "puppeteer",
   "engineOptions": {
-    "waitTimeout": 120000
+    "waitTimeout": 120000,
+    "args": [
+      "--no-sandbox"
+    ]
   },
-  "asyncCaptureLimit": 1,
+  "asyncCaptureLimit": 3,
```

Comment:
I wonder what is the actual effect of this change and if it really improves things. The reason is that the VMs used to run these tests are very small (only 2 vCPUs https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) their capacity to run things in parallel is very limited

Reply:
When using 3 async processes, the total time for taking screenshot is reduced from 8 to around 5-6 minutes. I tried with more than 3, but that fails, thats why I kept it at 3.